scrapy/commands/check.py

```diff
@@ -59,7 +59,7 @@ def add_options(self, parser):
     def run(self, args, opts):
         # load contracts
         contracts = build_component_list(self.settings.getwithbase('SPIDER_CONTRACTS'))
-        conman = ContractsManager(load_object(c) for c in contracts)
+        conman = ContractsManager([load_object(c) for c in contracts], opts.verbose)
```

Comment:
Any reason to turn the generator into a list?

Reply:
I was trying to keep it as a generator, but then it would have a syntax error once I add another argument(opts.verbose) to it. This is the easiest solution I have. Do you know if there is a better way to do this?